clients/admin-ui/cypress/e2e/systems-plus.cy.ts

```diff
@@ -159,4 +166,60 @@ describe("System management with Plus features", () => {
       });
     });
   });
+
+  describe("bulk system/vendor adding page", () => {
```

Comment:
I added some initial test coverage. It doesn't cover every possible permutation of functionality but it makes sure the page loads and the tables most important functions are working.

I didn't cover every possible scenario because I think this page is likely to be changed soon :sweat_smile: